scripts/monitoring/cron-send-project-count.sh

```diff
@@ -0,0 +1,13 @@
+!# /bin/bash
+
+#get the number of projects except which are ours
+projects=$(oca get projects | egrep -Ev 'NAME|default|openshift|openshift-infra' | wc -l)
+
+echo
+echo "Number of projects: $projects "
```

Comment:
In v3, we're not relying on cron e-mails to detect failures in our scripts, we're using ops-runner instead.

So, please have your script output data that makes sense by default, and we'll add things to ops-runner to make it nicer / quieter if we want.